### DISCUSSION TITLE
[Q&A] GLPI-Agent is renames screens

### DISCUSSION THREAD

**Discussion by waaalex** [glpi-project/glpi-agent#503] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hello,
When agent connects to glpi inventory, it works but it renames all our screens, even if we have renamed them by hand.
On computers or printers, no problems.

Is there a setting? Is it a bug?
Thanks.

Glpi 10.0.9
glpi-agetn 1.5
glpi-inventory plugin 1.2.3

**Comment by g-bougard** [comment]
Hi @waaalex 

just to fully understand from to which name your monitor was renamed ?

Anyway do you mean you entered them manually with a specific name before running computer inventory and so it is renamed on first inventory ?
If yes, you may need to lock the monitor name after running the first inventory by setting again its name. The lock will appear in the dedicated section:
![image](https://github.com/glpi-project/glpi-agent/assets/12514489/2d0938db-145a-46c0-947d-68662d2ca535)

**Comment by waaalex** [comment]
Hello,

**just to fully understand from to which name your monitor was renamed ?**
We have migrated from fusion agent to glpi agent.
Monitor was renamed from EC-XXX to LG 3250-BP

**Anyway do you mean you entered them manually with a specific name before running computer inventory and so it is renamed on first inventory ?**
Yes, After the first inventory from glpi agent, screens have been renamed. 
To be more precise, names were entered manually, before glpi agent migration.

Can we lock names massively (we have 500 screens ^^)?

Regards.

**Comment by waaalex** [comment]
Hello,
Another question, how to make a lock?
Regards.

**Comment by waaalex** [comment]
I found my answer.
https://github.com/glpi-project/glpi-inventory-plugin/issues/108
Thanks.
I will correct all my screen.
Regards.

**Comment by g-bougard** [comment]
Yes, a colleague reminded me about the global locks and where to find it: a top button on the inventory administration.
So you just need to add a global lock on monitor name. To have something like and don't forget to add a column to show the global attribut of a lock:
![image](https://github.com/glpi-project/glpi-agent/assets/12514489/9acd8dbb-af80-4b52-a453-7160e4674897)
